### PR TITLE
Breaks out the inject logic into another directive

### PR DESCRIPTION
**Pull request by BigFunger**

### CODE REVIEW

File: src/core_plugins/getting_started/public/components/getting_started/getting_started.html [hunk 1]
```diff
@@ -29,10 +29,9 @@ <h1 class="kuiTitle gettingStartedTitle">
     />
   </div>
 
-  <div
-    id="gettingStartedTopMessagesContainer"
-    class="kuiViewContentItem kuiVerticalRhythm"
-  ></div>
+  <div class="kuiCard__footer">
```
Comment (ycombinator) [inline src/core_plugins/getting_started/public/components/getting_started/getting_started.html]: This should be fixed to the classes on the left side.

**Review by ycombinator — CHANGES_REQUESTED** [review]
Just one class typo. After that LGTM!

**Comment by BigFunger** [comment]
@ycombinator Ready for your eyes again

File: src/core_plugins/getting_started/public/components/injected_items/injected_items.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import template from './injected_items.html';
 import './injected_items.less';
 
 function makeAngularParseableExpression(item) {
-  return `<div>${item}&nbsp;</div>`;
+  return `<div class="injected-items-item">${item}&nbsp;</div>`;
```
Comment (ycombinator) [inline src/core_plugins/getting_started/public/components/injected_items/injected_items.js]: I think per our [CSS styleguide](https://github.com/elastic/kibana/blob/master/style_guides/css_style_guide.md) this should be named something like `injectedItems__item`. @cjcenizal?

**Pull request merged**